ocm_python_wrapper/cluster.py

```diff
@@ -574,3 +590,11 @@ def check_param_conditions(clusters_dict, condition, condition_value):
             isinstance(condition_value, list)
             and cluster_condition_value in condition_value
         ) or cluster_condition_value == condition_value
+
+    @staticmethod
+    def addon_installation_instance_sampler(func, wait_timeout=TIMEOUT_30MIN):
```

Comment:
also why this function was changed?

Reply:
I originally thought to change it since the `TimeoutSampler` is now called [more than once](https://github.com/RedHatQE/openshift-cluster-management-python-wrapper/pull/55/files#diff-3368f738eecbcf02482808ff710a68f13e75401e49898e0e7fa6d0a2b9aefdd3R481):

- recive the func as arg to enable multiple usages 
- (TO DO) recive alsoo kwargs such as [command](https://github.com/RedHatQE/openshift-cluster-management-python-wrapper/pull/55/files#diff-3368f738eecbcf02482808ff710a68f13e75401e49898e0e7fa6d0a2b9aefdd3R387)

shelve or proceed?

Reply:
@myakove

Reply:
Suggestions added to recent commit